pkg/render/fluentd.go

```diff
@@ -257,9 +256,9 @@ func (c *fluentdComponent) path(path string) string {
 func (c *fluentdComponent) Objects() ([]client.Object, []client.Object) {
 	var objs, toDelete []client.Object
 	objs = append(objs, CreateNamespace(LogCollectorNamespace, c.cfg.Installation.KubernetesProvider, PSSPrivileged))
-	objs = append(objs, c.allowTigeraPolicy())
-	if c.cfg.EnterpriseVariant {
-		objs = append(objs, secret.ToRuntimeObjects(secret.CopyToNamespace(LogCollectorNamespace, c.cfg.PullSecrets...)...)...)
+	objs = append(objs, secret.ToRuntimeObjects(secret.CopyToNamespace(LogCollectorNamespace, c.cfg.PullSecrets...)...)...)
+	if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
```

Comment:
if we're not going to install this on OSS, we need to handle deleting this policy when downgrading from Enterprise -> OSS